todos/week1/3_string_manipulationtodo.py

```diff
@@ -2,15 +2,18 @@
 Personality questionnaire
 Referenced lesson(s): week_1/2_variables_and_types.py && week_1/3_string_manipulation.py
 """
-
+# TODO: Section 1:
 # TODO: Ask a user to input their full name and set it equal to a variable of `full_name`.
 # TODO: Greet the user by printing the statement "Hello _____, welcome to your personality quiz!"
 # TODO: Use a function from our last lesson to capitalize the user's name within the above print
 # TODO: statement. Test it's functionality by entering a name with lowercase letters.
 
+# TODO: #########################################################################################
+
+# TODO: Section 2:
 # TODO: Write a program that asks a user 6 unique questions about them. After receiving all of the
-# TODO: inputs, make a series of print statements that reflect the question. For example, if you ask
-# TODO: someone their favorite movie, the output should read "Your favorite movie is _____".
+# TODO: inputs, make a print statement in paragraph style that utilizes the .format function. Be 
+# TODO: sure to take advantage of the .title function in the print statement for names and words
+# TODO: that need to be capitalized! 
 
-# TODO: Print out the results in a neat format and make sure to capitalize the user's name with the
-# TODO: title function
+# TODO: #########################################################################################
```

Comment:
There needs to be a blank line at the end of this file. Also you don't need the 100 #s. Only use that as a divider.